pages.tr/linux/atool.md

```diff
@@ -0,0 +1,20 @@
+# atool
+
+> Çeşitli biçimlerdeki arşivleri yönetin.
+> Daha fazla bilgi için: <https://www.nongnu.org/atool/>.
+
+- Bir zip arşivindeki dosyaları listele:
+
+`atool --list {{path/to/archive.zip}}`
+
+- Bir tar.gz arşivini yeni bir alt dizine (veya yalnızca bir dosya içeriyorsa geçerli dizine) çıkart:
+
+`atool --extract {{path/to/archive.tar.gz}}`
+
+- İki dosyaya sahip yeni bir 7zip arşivi oluştur:
+
+`atool --add {{path/to/archive.7z}} {{path/to/file1}} {{path/to/file2}}`
```

Comment:
Please translate the placeholders (like `path/to/file`) for get this easier for non-English speakers.